hardware_interface/include/hardware_interface/internal/interface_manager.h

```diff
@@ -54,10 +54,8 @@ struct CheckIsResourceManager {
   template <typename C>
   static void callCM(typename std::vector<C*>& managers, C* result, typename C::resource_manager_type*)
   {
-    std::vector<typename C::resource_manager_type*> managers_in;
-    // we have to typecase back to base class
-    for(typename std::vector<C*>::iterator it = managers.begin(); it != managers.end(); ++it)
-      managers_in.push_back(static_cast<typename C::resource_manager_type*>(*it));
+    // We have to typecase back to base class
```

Comment:
I think the comment here was meant to say
```suggestion
    // We have to typecast back to base class
```

Reply:
Updated in 60b2f95